incubator/haproxy-ingress/templates/NOTES.txt

```diff
@@ -2,17 +2,6 @@ The haproxy-ingress controller has been installed.
 
 {{- if contains "NodePort" .Values.controller.service.type }}
 Get the application URL by running these commands:
-
-{{- if (not (empty .Values.controller.service.nodePorts.http)) }}
-  export HTTP_NODE_PORT={{ .Values.controller.service.nodePorts.http }}
-{{- else }}
-  export HTTP_NODE_PORT=$(kubectl --namespace {{ .Release.Namespace }} get services -o jsonpath="{.spec.ports[0].nodePort}" {{ template "haproxy-ingress.fullname" . }})
-{{- end }}
-{{- if (not (empty .Values.controller.service.nodePorts.https)) }}
-  export HTTPS_NODE_PORT={{ .Values.controller.service.nodePorts.https }}
-{{- else }}
-  export HTTPS_NODE_PORT=$(kubectl --namespace {{ .Release.Namespace }} get services -o jsonpath="{.spec.ports[1].nodePort}" {{ template "haproxy-ingress.fullname" . }})
-{{- end }}
```

Comment:
If these lines are removed,  which message will show for NodePort info ?

Reply:
Thank you for pointing that out. At first fix I thought to remove it entirely. Then I realized that some people will need HTTP_NODE_PORT and HTTPS_NODE_PORT info.
Realizing that, I fixed it in 991993eb8102abbc9c9fb7893b1f5f761a0959e9

What do you think?